client/src/components/QA/QACardAnswers.jsx

```diff
@@ -31,40 +31,80 @@ const QACardAnswers = ({ currentAnswers }) => {
   }, []);
 
   const APICallHelpful = (answerId) => {
-    axios.put(`/qa/answers/:answer_id/helpful`, {answer_id: answerId})
+    axios.put(`/qa/answers/:answer_id/helpful`, { answer_id: answerId })
       .then(info => console.log('info:', info))
       .catch(err => err);
   };
 
   const APICallReport = (answerId) => {
-    axios.put(`/qa/answers/:answer_id/report`, {answer_id: answerId})
+    axios.put(`/qa/answers/:answer_id/report`, { answer_id: answerId })
       .then(info => console.log('info:', info))
       .catch(err => err);
   };
 
-  return (
-    <div data-testid="qa-answers-div">
-      <div data-testid="qa-div-card-answers">
-        {answersToShow.map((answer, index) =>
-          <div className="qa-answers-external" key={index}>
-            <div className="qa-div-answers" ><b>A:</b> {answer.body}</div>
-            <p className="qa-footer"> by {answer.answerer_name}, {format((new Date(answer.date)), "MMMM dd, yyyy")}
-              <span className="qa-divider">|</span>
-              <span onClick={() => APICallHelpful(answer.id)}>Helpful? ({answer.helpfulness})</span>
-              <span className="qa-divider">|</span>
-              <span onClick={() => APICallReport(answer.id)}>Report</span>
-            </p>
-          </div>
-        )}
-        <button data-testid="qa-load-more" className="qa-load-more" onClick={() => {
-          setAnswersToShowLength(answersToShowLength + 2);
-          setAnswersToShow(answers.slice(0, answersToShowLength));
-        }}><b><span className="qa-load">[LOAD MORE ANSWERS]</span></b></button>
-      </div>
-    </div>
-  );
-};
+  // if there are no answers
```

Comment:
This all looks great. Not worth doing now, but for the future, I would add the conditions within these chunks, aside from `answersLength === 0`, bc that makes sense. I just see a lot of duplicated code that could be condensed. e.g.

```javascript
// if there are more answers than those displayed, show "load more" button
{answersLength >= (answersToShowLength - 1)) && (
  <button data-testid="qa-load-more" className="qa-load-more" onClick={() => {
    setAnswersToShowLength(answersToShowLength + 2);
    setAnswersToShow(answers.slice(0, answersToShowLength));
  }}><b><span className="qa-load">[LOAD MORE ANSWERS]</span></b></button>
)}
```